src/components/AssetManagementPanel/AssetManagementApplication/index.tsx

```diff
@@ -49,9 +51,9 @@ export const AssetManagementApplication = ({ tokenId, tokenRegistryAddress }: As
           account={account}
           onConnectToWallet={upgradeProvider}
           beneficiary={beneficiary}
-          approvedBeneficiary={approvedBeneficiary}
+          approvedBeneficiary={approvedBeneficiary || ""}
           holder={holder}
-          approvedHolder={approvedHolder}
+          approvedHolder={approvedHolder || ""}
```

Comment:
think no need fallback to empty string

Reply:
i can move them to where we have the function call but the i kind of need it because the typescript lint is throwing an error, apparently when initiated, it is undefined.